front/src/utils/uiCoreHelpers.ts

```diff
@@ -0,0 +1,23 @@
+type SelectFixedOption =
+  | {
+      label: string;
+      id: string;
+    }
+  | undefined;
+
+// Utility to create standardized options for <Select> components.
+// `getOptionLabel`: Displays the option or a placeholder if empty.
+// `getOptionValue`: Uses the option itself as the value.
+export const createStringSelectOptions = (opts: string[]) => ({
+  options: opts,
+  getOptionLabel: (option: string) => option || '',
```

Comment:
Don't think we need that

```suggestion
  getOptionLabel: (option: string) => option,
```